stdlib/atomic.mli

```diff
@@ -17,6 +17,8 @@
 
 (** Atomic references.
 
+    See '10 Memory model: The hard bits' in the manual.
```

Comment:
This is very useful. 

Optional: The absolute reference to the manual chapter will need to be kept in sync with any manual changes. Is there a way to do it better.

Reply:
We should add a similar reference for `stdlib/effect.mli`.

Reply:
I was trying to look for existing places that would have such references, but either I was in looking in the wrong place or there aren't any. For now I'll drop the number and just refer to the title, that should still be easy enough for the user to find (well ideally this would be a clickable link, maybe I need to emit some LaTex for that from ocamldoc I'll try to take a look how to do that)

Reply:
Ok, I've added this for the PDF and HTML manuals by manually emitting a '\ref' and 'a href' and pushed a commit to show how this can be done for the 'Atomic' module.
I'm not sure this is the right approach (first class support for this in ocamldoc/odoc might be better), but in the short term would you like me to add these manual links to the other modules too?

Reply:
Unfortunately, it is only possible to add references to the manual in the API  documentation in the PDF output. For the html, the issue is that the API documentation and manual html output are generated independently. It is still safe to link from the manual to the API documentation because the standard library has some backward compatibility constraint which means that we can be reasonably be certain than an link to `Stdlib.List` can be given a canonical address. This is not clearly the case for the manual. Consequently, I think that citing the manual section name is the best short term solution .

Reply:
I've removed the %latex/%html and added a note in the commit message explaining the limitation and that this can be revisited in the future.